src/main/kotlin/dev/cbyrne/kdiscordipc/data/activity/Activity.kt

```diff
@@ -87,7 +119,7 @@ fun Activity.largeImage(key: String, text: String? = null) {
     this.assets?.largeText = text
 }
 
-fun Activity.party(id: String, size: List<Int>) {
+fun Activity.party(id: String, size: Activity.Party.PartySize) {
```

Comment:
Can we change this to take in the 2 values instead of needing to pass an instance of `PartySize`?